Nich-Thompson/Project4
# Auth updated

The same PR as Wesley's, this time with some additional changes that updated the .gitignore and should let the code run on all our machines. 
To test: 
1. Install composer in your powershell
2. Create a database (in your powershell:)
   mysql -u root
   create database inspectietool;
3. Create a .env from .env.example
4. Make sure the name, user and password in your .env are correct
5. php artisan migrate:fresh
6. php artisan db:seed
7. php artisan serve
8. (open new powershell) php artisan key:generate
9. go to your local website and go to the login page
10. log in: user: admin@inspectietool.nl pass: password
